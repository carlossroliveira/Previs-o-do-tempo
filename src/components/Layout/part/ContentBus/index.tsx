// -------------------------------------------------
// Packages
// -------------------------------------------------
import React from 'react';
// -------------------------------------------------
// Components
// -------------------------------------------------
import { Button } from '../../../Button';
import { Input } from '../../../Input';
// -------------------------------------------------
// Styles
// -------------------------------------------------
import { ContainerSC } from './contentBusStyles';

export const ContentBus = (): JSX.Element => {
  return (
    <ContainerSC>
      <Input />
      <Button />
    </ContainerSC>
  );
};
