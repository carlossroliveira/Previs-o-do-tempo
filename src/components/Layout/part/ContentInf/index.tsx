// -------------------------------------------------
// Packages
// -------------------------------------------------
import React from 'react';
// -------------------------------------------------
// Components
// -------------------------------------------------
import { SwitchComponent } from '../../../SwitchComponent';
// -------------------------------------------------
// Styles
// -------------------------------------------------
import { ContainerSC, TitleSC } from './contentInfStyles';

export const ContentInf = (): JSX.Element => {
  return (
    <ContainerSC>
      <TitleSC>Tema</TitleSC>
      <SwitchComponent />
    </ContainerSC>
  );
};
