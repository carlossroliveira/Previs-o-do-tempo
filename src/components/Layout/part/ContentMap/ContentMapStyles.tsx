import styled, { keyframes } from 'styled-components';

export const TextAnimation = keyframes`
  to {
      opacity: 1;
      transform: initial;
  }
`;

export const ContainerSC = styled.section`
  grid-area: contentMap;
  box-shadow: rgba(0, 0, 0, 0.17) 0px -23px 25px 0px inset,
    rgba(0, 0, 0, 0.15) 0px -36px 30px 0px inset,
    rgba(0, 0, 0, 0.1) 0px -79px 40px 0px inset, rgba(0, 0, 0, 0.06) 0px 2px 1px,
    rgba(0, 0, 0, 0.09) 0px 4px 2px, rgba(0, 0, 0, 0.09) 0px 8px 4px,
    rgba(0, 0, 0, 0.09) 0px 16px 8px, rgba(0, 0, 0, 0.09) 0px 32px 16px;

  padding: 1rem;
  color: ${(props) => props.theme.color.primary};

  transform: translateX(+50px);
  animation: ${TextAnimation} 0.5s linear forwards;
`;

export const ContainerPrimarySC = styled.div`
  display: flex;
  flex-wrap: wrap;
  align-items: center;
  justify-content: space-around;

  border-bottom: 2px solid gray;
`;

export const PrimaryFirstCubeSC = styled.div`
  text-align: center;
  h1 {
    font-weight: 600;
  }
`;

export const PrimarySecondCubeSC = styled.div`
  max-width: 175px;
  height: 127px;
`;

export const PrimaryThirdCubeSC = styled.div`
  text-align: center;
`;

export const ContainerSecondarySC = styled.div`
  display: flex;
  flex-wrap: wrap;
  justify-content: space-around;

  text-align: center;
`;

export const SecondaryFirstCubeSC = styled.div``;

export const SecondarySecondCubeSC = styled.div`
  width: 85px;
  height: 55px;
  img {
    width: 7rem;
  }
`;
