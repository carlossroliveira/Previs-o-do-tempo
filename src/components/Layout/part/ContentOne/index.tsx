// -------------------------------------------------
// Packages
// -------------------------------------------------
import React from 'react';
// -------------------------------------------------
// Components
// -------------------------------------------------

// -------------------------------------------------
// Styles
// -------------------------------------------------
import { ContainerSC } from './contentOneStyles';
// -------------------------------------------------
// Types
// -------------------------------------------------

export const ContentOne = (): JSX.Element => {
  return <ContainerSC>CONTENTONE</ContainerSC>;
};
