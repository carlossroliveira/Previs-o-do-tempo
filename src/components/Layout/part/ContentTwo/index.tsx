// -------------------------------------------------
// Packages
// -------------------------------------------------
import React from 'react';
// -------------------------------------------------
// Components
// -------------------------------------------------

// -------------------------------------------------
// Styles
// -------------------------------------------------
import { ContainerSC } from './contentTwoStyles';
// -------------------------------------------------
// Types
// -------------------------------------------------

export const ContentTwo = (): JSX.Element => {
  return <ContainerSC>CONTENTTWO</ContainerSC>;
};
