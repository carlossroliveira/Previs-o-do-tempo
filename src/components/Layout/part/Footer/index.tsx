// -------------------------------------------------
// Packages
// -------------------------------------------------
import React from 'react';
// -------------------------------------------------
// Components
// -------------------------------------------------

// -------------------------------------------------
// Styles
// -------------------------------------------------
import { ContainerSC } from './footerStyles';
// -------------------------------------------------
// Types
// -------------------------------------------------

export const Footer = (): JSX.Element => {
  return (
    <ContainerSC>
      Animações Animações Animações Animações Animações Animações Animações
      Animações Animações Animações Animações Animações Animações Animações
      Animações Animações Animações Animações Animações Animações Animações
      Animações Animações Animações Animações Animações Animações Animações
      Animações Animações Animações Animações Animações Animações Animações
      Animações Animações Animações Animações Animações Animações{' '}
    </ContainerSC>
  );
};
