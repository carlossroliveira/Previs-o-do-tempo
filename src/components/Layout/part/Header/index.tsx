// -------------------------------------------------
// Packages
// -------------------------------------------------
import React from 'react';
// -------------------------------------------------
// Components
// -------------------------------------------------

// -------------------------------------------------
// Styles
// -------------------------------------------------
import { ContainerSC } from './headerStyles';
// -------------------------------------------------
// Types
// -------------------------------------------------

export const Header = (): JSX.Element => {
  return <ContainerSC>HEADER</ContainerSC>;
};
