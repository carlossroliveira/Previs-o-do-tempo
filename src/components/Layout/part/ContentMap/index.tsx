// -------------------------------------------------
// Packages
// -------------------------------------------------
import React from 'react';
// -------------------------------------------------
// Components
// -------------------------------------------------

// -------------------------------------------------
// Styles
// -------------------------------------------------
import { ContainerSC } from './ContentMapStyles';
// -------------------------------------------------
// Types
// -------------------------------------------------

export const ContentMap = (): JSX.Element => {
  return <ContainerSC>MAPA</ContainerSC>;
};
