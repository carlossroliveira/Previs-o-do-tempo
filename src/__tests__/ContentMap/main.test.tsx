// -------------------------------------------------
// Packages
// -------------------------------------------------
import React from 'react';

import 'jest-styled-components';
import '@testing-library/jest-dom';
import { render } from '@testing-library/react';
// -------------------------------------------------
// Components
// -------------------------------------------------
import { ContentMap } from '../../components/Layout/part/ContentMap';
// -------------------------------------------------
// Utils
// -------------------------------------------------
import WithThemeComponent from '../utils/withThemeComponent';

const ContentMapWithTheme = WithThemeComponent(ContentMap);

describe('ContentMap Testing', () => {
  it('should match snapshot', () => {
    const component = render(<ContentMapWithTheme />);
    expect(component).toMatchSnapshot();
  });
});
