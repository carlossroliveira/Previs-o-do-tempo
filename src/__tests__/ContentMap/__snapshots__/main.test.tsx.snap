// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentMap Testing should match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": .c0 {
  grid-area: contentMap;
  box-shadow: rgba(0,0,0,0.17) 0px -23px 25px 0px inset, rgba(0,0,0,0.15) 0px -36px 30px 0px inset, rgba(0,0,0,0.1) 0px -79px 40px 0px inset,rgba(0,0,0,0.06) 0px 2px 1px, rgba(0,0,0,0.09) 0px 4px 2px,rgba(0,0,0,0.09) 0px 8px 4px, rgba(0,0,0,0.09) 0px 16px 8px,rgba(0,0,0,0.09) 0px 32px 16px;
  padding: 1rem;
  color: #EEEEEE;
  -webkit-transform: translateX(+50px);
  -ms-transform: translateX(+50px);
  transform: translateX(+50px);
  -webkit-animation: eQudGy 0.5s linear forwards;
  animation: eQudGy 0.5s linear forwards;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: space-around;
  -webkit-justify-content: space-around;
  -ms-flex-pack: space-around;
  justify-content: space-around;
  border-bottom: 2px solid gray;
}

.c2 {
  text-align: center;
}

.c2 h1 {
  font-weight: 600;
}

.c3 {
  max-width: 175px;
  height: 127px;
}

.c4 {
  text-align: center;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-box-pack: space-around;
  -webkit-justify-content: space-around;
  -ms-flex-pack: space-around;
  justify-content: space-around;
  text-align: center;
}

<body>
    <div>
      <section
        class="c0"
      >
        <div
          class="c1"
        >
          <div
            class="c2"
          >
            <h1 />
            <p />
            <p />
          </div>
          <div
            class="c3"
          >
            <img
              src="http://assets.api.hgbrasil.com/weather/images/undefined.png"
            />
          </div>
          <div
            class="c4"
          >
            <h1>
              °
            </h1>
            <p />
          </div>
        </div>
        <div
          class="c5"
        />
      </section>
    </div>
  </body>,
  "container": .c0 {
  grid-area: contentMap;
  box-shadow: rgba(0,0,0,0.17) 0px -23px 25px 0px inset, rgba(0,0,0,0.15) 0px -36px 30px 0px inset, rgba(0,0,0,0.1) 0px -79px 40px 0px inset,rgba(0,0,0,0.06) 0px 2px 1px, rgba(0,0,0,0.09) 0px 4px 2px,rgba(0,0,0,0.09) 0px 8px 4px, rgba(0,0,0,0.09) 0px 16px 8px,rgba(0,0,0,0.09) 0px 32px 16px;
  padding: 1rem;
  color: #EEEEEE;
  -webkit-transform: translateX(+50px);
  -ms-transform: translateX(+50px);
  transform: translateX(+50px);
  -webkit-animation: eQudGy 0.5s linear forwards;
  animation: eQudGy 0.5s linear forwards;
}

.c1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: space-around;
  -webkit-justify-content: space-around;
  -ms-flex-pack: space-around;
  justify-content: space-around;
  border-bottom: 2px solid gray;
}

.c2 {
  text-align: center;
}

.c2 h1 {
  font-weight: 600;
}

.c3 {
  max-width: 175px;
  height: 127px;
}

.c4 {
  text-align: center;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-box-pack: space-around;
  -webkit-justify-content: space-around;
  -ms-flex-pack: space-around;
  justify-content: space-around;
  text-align: center;
}

<div>
    <section
      class="c0"
    >
      <div
        class="c1"
      >
        <div
          class="c2"
        >
          <h1 />
          <p />
          <p />
        </div>
        <div
          class="c3"
        >
          <img
            src="http://assets.api.hgbrasil.com/weather/images/undefined.png"
          />
        </div>
        <div
          class="c4"
        >
          <h1>
            °
          </h1>
          <p />
        </div>
      </div>
      <div
        class="c5"
      />
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
